### ISSUE TITLE
Пустое окно при публикации копии поста

### ISSUE THREAD

**Issue by nikolaev-rd** [Telepost-me/support#35]
**Описание**
При создании копии поста и её публикации, вылезает пустое окно. 

**Как повторить**
1. Копируем уже опубликованный пост, правим его.
2. Пытаемся опубликовать пост снова.
3. Получаем пустое pop-up окно, которое нельзя закрыть.
   ![image](https://user-images.githubusercontent.com/24430718/119563782-ad0aec00-bdb0-11eb-890a-8ce85bfd13fb.png)
4. URL страницы ссылается на черновик, поэтому при обновлении этой страницы открывается редактирование только что опубликованного поста.

**Ожидаемое поведение**
Копия поста должна публиковаться как обычный пост.

**Важные детали:**
 - Тип поста: любой
 - Операционная система: любая
 - Браузер и его версия: Chrome v90.0.4430.212
 - Версия Telepost (web):  v2.3.1-1afc47ad

**Дополнительно**
Ссылка на обсуждение: https://t.me/c/1306965784/23663

**Comment by telepost-support** [comment]
Должно работать в 2.4, нужно перепроверить

**Comment by nikolaev-rd** [comment]
Да, при попытке воспроизвести по описанным шагам на версии v2.4.4-470d9ca4 — все работает как ожидается.

**Comment by nikolaev-rd** [comment]
Бага снова [ожила](https://t.me/c/1306965784/28330) (v2.5.0-f2b2186e): 
![telegram-cloud-photo-size-2-5269598150191069023-y](https://user-images.githubusercontent.com/24430718/130565827-20788ef0-cff3-49ed-81ac-59120adfeac1.jpg)
